resources/views/gatekeeper/profile.blade.php

```diff
@@ -39,7 +39,9 @@
                         @if (count($trainers)>0)
                            <h5 style="margin-bottom:0px;">Trainers</h5>   
                            @foreach($trainers as $trainer)
-                              <a href="/members/{{ $trainer->user()->first()->id }}/profile" title="View Profile"><span class="badge badge-primary badge-large">{{ $trainer->user()->first()->get_name() }}</span></a>&nbsp;
+                              @if ($trainer->user()->first()->status == 'active') 
```

Comment:
I think this will lead to some weirdness if trainers is 1 but inactive. I'd probably push for adding this logic at line 38 in the query, and failing that a filter:

```
$trainers = $gatekeeper->trainers()->get()->filter(function ($trainer) {
        return $trainer->user()->first()->status == 'active';
});
```

Reply:
Perfect. I knew fixing the query was the answer but I didn't know how to get the right query out of the ORM.

Reply:
I refactored to use (I believe) database query level relation attribute filtering.